queries.sql

```diff
@@ -1,3 +1,8 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01/01/2016' AND '01/01/2019';
+SELECT * from animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
```

Comment:
- [ ] Kindly add a query and screenshot that lists name and escape attempts of animals that weigh more than 10.5kg as this is a requirement.